OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,22 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.45
+17th June,2025
+
+## Core
+* Added the {{node_ips}} option to monitor nodes for events. Setting this field limits alerts monitoring to the nodes specified in the list. For example: ["127.0.0.1","127.0.0.2"]. (OPSC-17644)
+* *Targeted Solr Core Fetching:* OpsCenter now optimizes how Solr core information is retrieved. Instead of fetching cores for _all_ keyspaces and tables, OpsCenter will now fetch only the cores associated with the {{selected keyspaces and tables}}. This improves performance and reduces unnecessary data processing. (OPSC-17704)
+* Added enhancements to authentication configuration and user lockout management. (OPSC-15418)
+* Upgraded the {{org.json:json}} library to version {{20230227}}. (OPSC-17628 OPSC-17629)
+* Added new options to the {{authentication}} section of {{opscenterd.conf}} to allow users to use {{password_db}} and {{passwd_db}} alias. (OPSC-17701)
+* Improved data insertion logic to handle duplicates, enhancing reliability and preventing insertion errors. (OPSC-17693)
```

Comment:
What's the feature? A customer would have no idea what this is referring to. Add  "user role"  somewhere in there.